sematic/ui/packages/main/src/runs/RunIndex.tsx

```diff
@@ -144,8 +144,10 @@ const TableColumns: Array<RunListColumn> = [
 ];
 
 export function RunIndex() {
+    const params = new URLSearchParams(document.location.search);
+    const defaultSearchString = params.get("search");
     const [searchString, setSearchString] = useState<string | undefined>(
```

Comment:
We can simply set the default value to "" and deal with the deep link info in the useEffect.